src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -108,18 +117,71 @@ public function getVerticalBorderChar()
         return $this->verticalBorderChar;
     }
 
+    /**
+     * Sets crossing characters.
+     *
+     * Example:
+     * <code>
+     * 1---------------2-----------------------2------------------3
+     * | ISBN          | Title                 | Author           |
+     * 8---------------0-----------------------0------------------4
+     * | 99921-58-10-7 | Divine Comedy         | Dante Alighieri  |
+     * | 9971-5-0210-0 | A Tale of Two Cities  | Charles Dickens  |
+     * | 960-425-059-0 | The Lord of the Rings | J. R. R. Tolkien |
+     * 7---------------6-----------------------6------------------5
+     * </code>
+     *
+     * @param string $cross       Crossing char (see #0 of example)
+     * @param string $topLeft     Top left char (see #1 of example)
+     * @param string $topMid      Top mid char (see #2 of example)
+     * @param string $topRight    Top right char (see #3 of example)
+     * @param string $midRight    Mid right char (see #4 of example)
+     * @param string $bottomRight Bottom right char (see #5 of example)
+     * @param string $bottomMid   Bottom mid char (see #6 of example)
+     * @param string $bottomLeft  Bottom left char (see #7 of example)
+     * @param string $midLeft     Mid left char (see #8 of example)
+     *
+     * @return $this
+     */
+    public function setCrossingChars(string $cross, string $topLeft, string $topMid, string $topRight, string $midRight, string $bottomRight, string $bottomMid, string $bottomLeft, string $midLeft): self
+    {
+        $this->crossingChar = $cross;
+        $this->crossingTopLeftChar = $topLeft;
+        $this->crossingTopMidChar = $topMid;
+        $this->crossingTopRightChar = $topRight;
+        $this->crossingMidRightChar = $midRight;
+        $this->crossingBottomRightChar = $bottomRight;
+        $this->crossingBottomMidChar = $bottomMid;
+        $this->crossingBottomLeftChar = $bottomLeft;
+        $this->crossingMidLeftChar = $midLeft;
+
+        return $this;
+    }
+
+    /**
+     * Sets default crossing character used for each cross.
+     *
+     * @see {@link setCrossingChars()} for setting each crossing individually.
+     */
+    public function setDefaultCrossingChar(string $char): self
+    {
+        return $this->setCrossingChars($char, $char, $char, $char, $char, $char, $char, $char, $char);
+    }
+
     /**
      * Sets crossing character.
      *
      * @param string $crossingChar
      *
      * @return $this
+     *
+     * @deprecated since Symfony 4.1, to be removed in 5.0. Use {@link setDefaultCrossingChar()} instead.
```

Comment:
`{@link method()}` renders OK after compile? not sure it's needed / consistent:

https://github.com/symfony/symfony/search?l=PHP&q=%40deprecated+use+instead&type=&utf8=%E2%9C%93

Reply:
Looks good :) and often used in the code base.
![screenshot from 2018-03-12 10-35-25](https://user-images.githubusercontent.com/4578773/37260659-84175e7e-25e1-11e8-9f4e-bd807193bc2d.png)

Reply:
(annex comment: `{@link method()}` is not rendered on api.symfony.com. E.g. https://api.symfony.com/3.4/Symfony/Bundle/SecurityBundle/Security/FirewallContext.html#method_getContext)

Reply:
I would remove the @link here as we've never done that before elsewhere AFAIK.

Reply:
It's quite used in fact (e.g. on 3.4, more than 300 occurrences those 13 in a `@deprecated` annotation) and it makes sense here IMO. Sure?

Reply:
ok let's keep like this then :)